src/graphApp.test.js

```diff
@@ -0,0 +1,325 @@
+    expect(newY_points).toHaveLength(1)
+    
+    expect(newX_points[0]).not.toHaveValue()
+    expect(newY_points[0]).not.toHaveValue()
+
+})
+
+test("Clearing the data button resets color picker", async function() {
+	// Arrange:
+	initDomFromFIles(`${__dirname}/line/line.html`, `${__dirname}/line/line.js`)
+
+    //Acquire Chart title
+    const Clear_chart_data_btn = domTesting.getByRole(document, "button", {name : "Clear chart data"})
+    const Chart_color_btn = domTesting.getByLabelText(document, "Chart color")
+
+    //Acquire Chart title
+    domTesting.fireEvent.input(Chart_color_btn, {target: {value: '#00ff4C'}})
+
+	// Act:
+	const user = userEvent.setup()
+	await user.click(Clear_chart_data_btn)
+    await user.click(Chart_color_btn)
+    await user.click(Chart_color_btn)
+	// Assert:
+    expect(Chart_color_btn).toHaveValue("#ff4500")
+
+})
+
+})
+
+test("Sending correct values to generate graph function", async function() {
+	// Arrange:
+	initDomFromFIles(`${__dirname}/line/line.html`, `${__dirname}/line/line.js`)
+
+    const generateChartImg = require("./lib/generateChartImg.js")
+    jest.mock('./lib/generateChartImg.js');
+    generateChartImg.mockImplementation((type, data, xLabel, yLabel, title, color) => {})
+    
+    //Aquire buttons/inputs
+    const Chart_color_btn = domTesting.getByLabelText(document, "Chart color")
+    const generateChartButton = domTesting.getByRole(document, "button", {name : "Generate chart"})
+    const add_Point_btn = domTesting.getByRole(document, "button", {name : "+"})
+    const Chart_title_input = domTesting.getByLabelText(document, "Chart title")
+
+    const X_Axis_title = domTesting.getByLabelText(document, "X label")
+    const Y_Axis_title = domTesting.getByLabelText(document, "Y label")
+
+    // Create more points
+	const user = userEvent.setup()
+	await user.click(add_Point_btn)
+    await user.click(add_Point_btn)
+
+    //Get points inputs
+    const X_points = domTesting.queryAllByLabelText(document, "X")
+    const Y_points = domTesting.queryAllByLabelText(document, "Y")
+
+    //Create fake inputs
+    domTesting.fireEvent.input(Chart_color_btn, {target: {value: '#00ff4C'}})
+
+    await user.type(Chart_title_input, "Test title")
+    await user.type(X_Axis_title, "X axis title")
+    await user.type(Y_Axis_title, "Y axis title")
+
+    await user.type(X_points[0], "1")
+    await user.type(Y_points[0], "2")
+    await user.type(X_points[1], "3")
+    await user.type(Y_points[1], "4")
+    await user.type(X_points[2], "5")
+    await user.type(Y_points[2], "6")
+
+    await user.click(generateChartButton)
+
+    //Make generate chart went through
+    expect(generateChartImg).toHaveBeenCalledTimes(1)
+    expect(generateChartImg).toHaveBeenCalledWith
+    (
+        "line",
+         [{"x": "1", "y": "2"},{"x": "3", "y": "4"},{"x": "5", "y": "6"}], 
+          "X axis title",
+          "Y axis title",
+          "Test title",
+          "#00ff4c"
+    )
+
```

Comment:
All of the test look good and are very well written. Only thing i'd change is what brian mentioned